leaks_test.go

```diff
@@ -83,6 +84,57 @@ func TestVerifyNone(t *testing.T) {
 	bg.unblock()
 }
 
+func TestIgnoreCurrent(t *testing.T) {
+	t.Run("Should ignore current", func(t *testing.T) {
+		defer VerifyNone(t)
+		done := make(chan struct{})
+		go func() {
+			<-done
+		}()
+		VerifyNone(t, IgnoreCurrent())
+		close(done)
+	})
+
+	t.Run("Should detect new leaks", func(t *testing.T) {
+		done := make(chan struct{})
+		defer VerifyNone(t)
+		defer close(done)
+		defer func(o Option) {
+			err := Find(o)
+			require.Error(t, err)
+		}(IgnoreCurrent())
+		go func() {
+			<-done
+		}()
+	})
+
+	t.Run("Should not ignore false positive", func(t *testing.T) {
+		defer VerifyNone(t)
+		const goroutinesCount = 5
+		wg := sync.WaitGroup{}
+		done := make(chan struct{})
+		for i := 0; i < goroutinesCount; i++ {
+			wg.Add(1)
+			go func() {
+				<-done
+				wg.Done()
+			}()
+		}
```

Comment:
can you add some whitespace to this test to separate out the different parts, and add comments on what we are trying to test?

Reply:
> can you add some whitespace to this test to separate out the different parts, and add comments on what we are trying to test?

Sure, done. Please take a look :)